### DISCUSSION TITLE
UI scaling on Gnome Wayland Linux?

### DISCUSSION THREAD

**Discussion by HubKing** [dbeaver/dbeaver#20165] [Q&A]
Is there a simply way to zoom in the entire UI, like Blender or QT apps?

I see there is font settings in Preferences, but there are too many items to manually configure. And from my experience, just changing the font size can often make UI funky, because the control/dialogue sizes were configured to fit the default font size.

I have tried `dbeaver -Dswt.autoScale=400`, but it had no effect.

**Comment by Aneeqasif** [comment]
> Is there a simply way to zoom in the entire UI, like Blender or QT apps?
> 
> I see there is font settings in Preferences, but there are too many items to manually configure. And from my experience, just changing the font size can often make UI funky, because the control/dialogue sizes were configured to fit the default font size.
> 
> I have tried `dbeaver -Dswt.autoScale=400`, but it had no effect.

did you find any solution?
